scripts/plot_dispatch.py

```diff
@@ -141,8 +145,40 @@
             ax.set_position(
                 [box.x0, box.y0 + box.height * 0.15, box.width, box.height * 0.85]
             )
+
+            # Simplify legend. As there is only one color per technology, there should
+            # be only one label per technology.
+            simple_labels_dict = {
+                "Battery": ["Battery out", "Battery in"],
+                "El. transmission external": ["El. import", "El. export"],
+                "El. transmission B-BB": [
+                    "El. transmission in",
+                    "El. transmission out",
+                ],
+                "El. shortage / curtailment": ["El. shortage", "Curtailment"],
+                "Heat cen. storage": ["Heat cen. storage out", "Heat cen. storage in"],
+                "Heat cen. mismatch": ["Heat cen. excess", "Heat cen. shortage"],
+                "Heat dec. storage": ["Heat dec. storage out", "Heat dec. storage in"],
+                "Heat dec. mismatch": ["Heat dec. excess", "Heat dec. shortage"],
+            }
+            handles, labels = ax.get_legend_handles_labels()
+
+            for key, value in simple_labels_dict.items():
+                if value[0] in labels and value[1] in labels:
+                    labels = [
+                        key
+                        if item == value[0]
+                        else "_Hidden"
+                        if item == value[1]
+                        else item
+                        for item in labels
+                    ]
```

Comment:
Could you make this a function or write an inline comment to make it easier to understand?